packages/backend/src/queryBuilder.ts

```diff
@@ -69,19 +69,25 @@ export const replaceUserAttributes = (
         const userAttribute = userAttributes.find(
             (ua) => ua.name === attribute,
         );
+
         if (userAttribute === undefined) {
             throw new ForbiddenError(
                 `Missing user attribute "${attribute}" on ${filter}: "${sqlFilter}"`,
             );
         }
-        if (userAttribute.users.length !== 1) {
+        if (userAttribute.users.length !== 1 && !userAttribute.default) {
```

Comment:
Should we check explicitly for null, in case we later want to support falsy values for the default?